.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.1.0
+  hooks:
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+  - id: check-yaml
+  - id: check-added-large-files
+- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+  rev: v2.2.0
+  hooks:
+  - id: pretty-format-yaml
+    args: [--autofix]
+- repo: https://github.com/pycqa/isort
+  rev: 5.10.1
+  hooks:
+  - id: isort
+    name: isort (python)
+    args: [--profile=black]
+    exclude: test_data/samples/.*.py
+- repo: https://github.com/psf/black
+  rev: 22.1.0
+  hooks:
+  - id: black
+    exclude: test_data/samples/.*.py
```

Comment:
You forgot `ssort`. :D

Reply:
on black hook exclude?

Reply:
After the black hook.  Run ssort from `local` repo.

Reply:
unfortunately it would make the hook more complicated because pre commit have to install repo first before run hook,  

so i use pre commit hook rather than run repo locally 

this is a bit different with tox hook because it assume tox already installed when running local hook